corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html

```diff
@@ -49,6 +50,31 @@
                 )
             }
 
+            this.dataType = ko.computed(function() {
+                // Returns type of Question, applicable only if optionsContainQuestions is true
+                var property = this.property()
+                var propertyObject = _.find(this.propertyOptions(), function(option){
+                    return option.value == property;
+                });
+                if (propertyObject !== undefined && propertyObject.hasOwnProperty("type"))
+                    return propertyObject.type;
+                else
+                    return undefined;
+            }, this);
+
+            this.calculationOptions = ko.observableArray(defaultCalculationOptions);
+            if (this.optionsContainQuestions) {
+                // If options are questions, calculationOptions should be limited based on the dataType
+                this.calculationOptions = ko.computed(function() {
+                    if (this.dataType() == "Int") {
```

Comment:
I think we need to check for a few more types here. I think they are `"Int"`, `"Double"`, and `"Long"`. We also want to allow any aggregation type on hidden value questions, because those aren't necessarily non-numeric.